pulp_smash/utils.py

```diff
@@ -9,6 +9,7 @@
 from packaging.version import Version
 from pulp_smash.constants import REPOSITORY_PATH, USER_PATH
 from time import sleep
+from urlparse import urljoin
```

Comment:
Pure addition of path parts is dangerous, because standard "urljoin" logic, which follows rules in various RFCs, behaves differently. For example:

```
>>> '/a/b/c' + '/d/e'
'/a/b/c/d/e'
>>> urljoin('/a/b/c', '/d/e')
'/d/e'
```

Pulp should produce paths that behave correctly with urljoin, and we've had problems in the past with developers making "relative paths" that have a leading slash.

I expect most clients of our REST API will use a urljoin implementation to combine paths.

Do you prefer that I undo this use of urljoin and file an issue to change them all at once? Or can we leave this as an incremental improvement, and file an issue to change the rest?